pytorch_lightning/accelerators/accelerator.py

```diff
@@ -222,6 +219,18 @@ def __setstate__(self, d):
     def on_save(self, checkpoint):
         return checkpoint
 
+    @property
+    def rpc_enabled(self):
+        return self.ddp_plugin is not None and isinstance(self.ddp_plugin, RPCPlugin)
+
+    @property
+    def distributed_sampler_kwargs(self):
+        raise NotImplementedError
+
+    @property
+    def require_distributed_sampler(self):
+        raise NotImplementedError
```

Comment:
Same as above. Can we make it return false per default?

Reply:
I think it is fine for both. An accelerator shouldn't be used. It is metaclass.